contrib/pylightning/lightning/lightning.py

```diff
@@ -33,36 +32,41 @@ def _readobj(self, sock):
                 pass
 
     def __getattr__(self, name):
-        """Intercept any call that is not explicitly defined and call _call
+        """Intercept any call that is not explicitly defined and call @call
 
         We might still want to define the actual methods in the subclasses for
         documentation purposes.
         """
         name = name.replace('_', '-')
 
         def wrapper(*args, **_):
-            return self._call(name, args)
+            return self.call(name, args)
         return wrapper
 
-    def _call(self, method, args=None):
-        logging.debug("Calling %s with arguments %r", method, args)
+    def call(self, method, payload=None):
+        self.logger.debug("Calling %s with payload %r", method, payload)
 
         sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
         sock.connect(self.socket_path)
         self._writeobj(sock, {
             "method": method,
-            "params": args or (),
+            "params": payload or {},
```

Comment:
Optionally, we can filter None values from this dict here to preserve existing behavior and avoid foo: null in JSON.